addons/bus/websocket.py

```diff
@@ -572,7 +572,13 @@ def _handle_transport_error(self, exc):
             code = CloseCode.SESSION_EXPIRED
         if code is CloseCode.SERVER_ERROR:
             reason = None
-            _logger.error(exc, exc_info=True)
+            registry = Registry(self._session.db)
+            sequence = registry.registry_sequence
+            registry = registry.check_signaling()
+            if (sequence != registry.registry_sequence) or not registry.loaded:
```

Comment:
Two things are wrong here:
1/ you inverted ERROR & WARNING. Warning should be when it failed because of a registry loading.
2/ I would remove "or not registry.loaded" (I think it's not useful as the bus does not reload registries, and it fail as you test the new registry, not the old one)